c2_utils/src/mfx_defs.cpp

```diff
@@ -81,6 +81,7 @@ mfxStatus MFXLoadSurfaceSW(uint8_t *data, uint32_t stride, const mfxFrameInfo& i
     if (!MFX_C2_IS_COPY_NEEDED(srf->Data.MemType, input_info, srf->Info)) {
         srf->Data.Y  = data;
         srf->Data.UV = data + nOWidth * nOHeight;
+        srf->Data.V = srf->Data.UV + 1;
```

Comment:
if use system memory, one vpl will check V statue.

`
    if (video.IOPattern == MFX_IOPATTERN_IN_SYSTEM_MEMORY)
    {
        MFX_CHECK(!LumaIsNull(&surface) || surface.Data.MemId, MFX_ERR_NULL_PTR);
        if (surface.Info.FourCC != MFX_FOURCC_Y410)
        {
            MFX_CHECK(surface.Data.U != 0 || (surface.Data.MemId && LumaIsNull(&surface)), MFX_ERR_NULL_PTR);
            MFX_CHECK(surface.Data.V != 0 || (surface.Data.MemId && LumaIsNull(&surface)), MFX_ERR_NULL_PTR);
        }
    }
`

Reply:
Please keep the original statement, otherwise srf->data.UV is null.
srf->Data.UV = data + nOWidth * nOHeight;